api/v2/controllers/bouquets/bouquets_controller.py

```diff
@@ -33,3 +33,21 @@ def post(self):
                                   refresh_token=credentials['refresh_token'])
         return make_response(jsonify({'data': {'message': 'successfully added bouquet',
                                                'bouquet': data}}), 201)
+    
+    def delete(self):
+        bouquet_id = request.args['bouquet_id']
+
+        try:
+            if bouquet_id.isdigit():
+                one_bouquet = query_bouquet(bouquet_id)
+                if one_bouquet:
+                    BouquetsModel.delete_bouquet(BouquetsModel, bouquet_id)
+                    return make_response(jsonify({'message': 'The bouquet was deleted successfully'}), 200)
+                
+                return make_response(jsonify({'message': 'The bouquet you want to delete is not found'}), 404)
+            
```

Comment:
You've got too much unnecessary blank lines. Please read the PEP 8 recommendation 👉 https://www.python.org/dev/peps/pep-0008/#blank-lines

Blank lines are to be used sparingly.